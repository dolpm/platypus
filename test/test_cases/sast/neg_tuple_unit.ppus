pipe main [] |> unit {
  [unit * int] schmunit = tuple((), 10);

  |> ();
}
