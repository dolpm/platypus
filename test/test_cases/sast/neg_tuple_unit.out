Fatal error: exception Failure("Illegal type. Tuples cannot contain units.")
